grpc_server.go

```diff
@@ -373,7 +373,7 @@ func (s *server) RealTime(stream pb.SpaceCloud_RealTimeServer) error {
 
 			// Subscribe to relaitme feed
 			feedData, err := s.realtime.Subscribe(ctx, clientID, s.auth, s.crud, data, func(feed *model.FeedData) {
-				client.Write(&model.Message{Type: utils.TypeRealtimeFeed, Data: feed})
+				client.Write(&model.Message{ID: req.ID, Type: utils.TypeRealtimeFeed, Data: feed})
```

Comment:
Why do we need to send the id here?

Reply:
Just to be extra sure that we are responding to the correct client

Reply:
Okay. I don't think that is required. The client receiving the message is identified by the `client` object we are calling write on

Reply:
I am checking for the id at the client side. Serves as a double check. Like how we check during the Subscribe response. As it is, we have the id field empty. Lets use it. 😜

Reply:
Lol. Okay